### PR TITLE
Adds support for setting system properties.

### PR DESCRIPTION
**Pull request by mrunalp**
This allows one to do the equivalent of sysctl -w ip.ipv4.ip_forward 1 or any such property.

I will add tests and other bits if we are okay adding this.
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by hqhq** [comment]
Seems risky if we allow users to set all system properties.

**Comment by mrunalp** [comment]
@hqhq docker could filter what is allowed. In libcontainer, it makes sense to be flexible.

**Comment by hqhq** [comment]
Right, +1 for me then, thanks.

**Comment by LK4D4** [comment]
Looks cool

**Comment by mrunalp** [comment]
Updated with tests.

**Comment by mrunalp** [comment]
@LK4D4 @crosbymichael PTAL.

**Comment by crosbymichael** [comment]
LGTM

File: rootfs_linux.go [hunk 1]
```diff
@@ -387,3 +388,12 @@ func maskFile(path string) error {
 	}
 	return nil
 }
+
+// writeSystemProperty writes the value to a path under /proc/sys as determined from the key.
+// For e.g. net.ipv4.ip_forward translated to /proc/sys/net/ipv4/ip_forward.
+func writeSystemProperty(key, value string) error {
+	path_prefix := "/proc/sys"
```
Comment (crosbymichael) [inline rootfs_linux.go]: camelCase

File: rootfs_linux.go [hunk 2]
```diff
@@ -387,3 +388,12 @@ func maskFile(path string) error {
...
+	key_path := strings.Replace(key, ".", "/", -1)
+	full_path := path.Join(path_prefix, key_path)
```
Comment (crosbymichael) [inline rootfs_linux.go]: Could probably reduce the amount of vars in this func as well

File: rootfs_linux.go (hunk 1 above)
Comment (mrunalp) [inline rootfs_linux.go]: Fixed.

File: rootfs_linux.go (hunk 2 above)
Comment (mrunalp) [inline rootfs_linux.go]: Fixed.

**Comment by LK4D4** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**